samples/benchmarks/coremark/README.rst

```diff
@@ -51,6 +51,7 @@ The sample configuration sets up the following board targets for standard loggin
 * ``nrf54l15dk/nrf54l05/cpuapp``
 * ``nrf54l15dk/nrf54l10/cpuapp``
 * ``nrf54l15dk/nrf54l15/cpuapp``
+* ``nrf54lm20pdk/nrf54lm20a/cpuapp``
```

Comment:
We shouldn't have mention of nRF54LM20 in docs.
cc: @annwoj.

Reply:
Ok, removed it then :)